Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>module com.example.crimealert {\r\n    requires javafx.controls;\r\n    requires javafx.fxml;\r\n\r\n    requires org.controlsfx.controls;\r\n    requires com.dlsc.formsfx;\r\n    requires org.kordamp.bootstrapfx.core;\r\n\r\n    opens com.example.crimealert to javafx.fxml;\r\n//    exports com.example.crimealert;\r\n    exports com.example.crimealert.view;\r\n    opens com.example.crimealert.view to javafx.fxml;\r\n    exports com.example.crimealert.controllor;\r\n    opens com.example.crimealert.controllor to javafx.fxml;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	(revision 42ffe3e8bae5e51df7e57d7dda79d8c0e4abf090)
+++ b/src/main/java/module-info.java	(date 1700936920743)
@@ -5,6 +5,8 @@
     requires org.controlsfx.controls;
     requires com.dlsc.formsfx;
     requires org.kordamp.bootstrapfx.core;
+    requires java.sql;
+    requires java.desktop;
 
     opens com.example.crimealert to javafx.fxml;
 //    exports com.example.crimealert;
